packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -343,63 +349,64 @@ const Dropdown = ({
-                        className={dropdownDisplayClassName()}
-                        tabIndex={isSingleOption() ? '-1' : '0'}
-                        onClick={handleVisibility}
-                        onKeyDown={onKeyPressed}
-                        id='dropdown-display'
-                        ref={dropdown_ref}
+                        className={classNames('dc-dropdown__container', {
+                            'dc-dropdown__container--suffix-icon': suffix_icon,
+                        })}
                     >
-                        <DisplayText
-                            has_symbol={has_symbol}
-                            name={name}
-                            is_title={is_list_visible}
-                            placeholder={placeholder}
-                            value={value || 0}
-                            list={list}
-                            is_align_text_left={is_align_text_left}
-                        />
-                    </div>
-                    {!isSingleOption() && (
-                        <Icon
-                            icon={is_alignment_left ? 'IcChevronLeft' : 'IcChevronDown'}
-                            className={classNames('dc-dropdown__select-arrow', {
-                                'dc-dropdown__select-arrow--left': is_alignment_left,
-                                'dc-dropdown__select-arrow--up': is_list_visible,
-                                'dc-dropdown__select-arrow--error': error || hint,
-                            })}
-                        />
-                    )}
-                    {error && <p className='dc-field--error'>{error}</p>}
-                    {is_nativepicker ? (
-                        <NativeSelect
-                            ref={native_select_ref}
-                            name={name}
-                            value={value}
-                            list={list}
-                            onChange={onChange}
-                        />
-                    ) : (
+                        {label && (
+                            <span
+                                className={classNames('dc-dropdown__label', {
+                                    'dc-dropdown__label--clicked': is_list_visible,
+                                })}
+                            >
+                                {label}
+                            </span>
+                        )}
+                        <div
+                            className={dropdownDisplayClassName()}
+                            tabIndex={isSingleOption() ? '-1' : '0'}
+                            onClick={handleVisibility}
+                            onKeyDown={onKeyPressed}
+                            id='dropdown-display'
+                            ref={dropdown_ref}
+                        >
+                            {suffix_icon && <Icon className='suffix-icon' icon={suffix_icon} size={16} fill />}
+                            <DesktopWrapper>
```

Comment:
What's the reason we have this `DesktopWrapper` nested into another `DesktopWrapper`?